web/packages/quick-editor/src/index.ts

```diff
@@ -0,0 +1,78 @@
+export type GravatarQuickEditorOptions = {
+	avatarSelector?: string;
+	scope?: string[];
+}
+
+export default class GravatarQuickEditor {
+	_triggerElement: HTMLElement;
+	_avatarElements: NodeListOf<HTMLImageElement>;
+	_email: string;
+	_scope: string[];
+
+	constructor( triggerSelector: string, email: string, options: GravatarQuickEditorOptions ) {
+		this._triggerElement = document.querySelector( triggerSelector );
+		this._email = email;
+
+		if ( ! this._triggerElement ) {
+			console.error( 'Gravatar Quick Editor: Trigger element not found' );
```

Comment:
I’m concerned this might be outputted by the developer’s site or library. From a DX perspective, ideally, error logs should only be displayed in the development build (as shown below). Unfortunately, we haven't handled this case yet 😅.

```js
if ( 'production' !== process.env.NODE_ENV ) {
  console.error( '...' );
}
```

Reply:
I have removed some of the error messages that may be unnecessary.
But I don't know if we should show them only on the developer build.
There are situations where a developer could get a hint of what's going wrong in their site if a user mentions an error message in the console.